locksmithctl/daemon.go

```diff
@@ -116,6 +116,50 @@ func expBackoff(interval time.Duration) time.Duration {
 	return interval
 }
 
+func getPeriodicAndSleep() {
```

Comment:
even if the function is not public it's still good to comment it
```suggestion
// getPeriodicAndSleep ...
func getPeriodicAndSleep() {
```

Reply:
This means I have to add a small description of the function in the comments?

Reply:
yes, exactly ! See this [blog](https://blog.golang.org/godoc) article :)